README.md

```diff
@@ -110,11 +95,9 @@ resource is set for internal consumption the user shouldn't have to know it.
   }
 ```
 
-For the web components we collect all resources assigned to nodes in the
-application. We then use the map function to loop over them declaring the
-components and collecting the Http resources they export for later consumption.
-This allows the user to declare one or more web components to scale their
-application dynamically through the declaration.
+The load balancer component of the application is optional, and you may have any number of instances of it. In some cases, you may have no need for a load balancer, or you may require several in a high availability configuration. Each load balancer requires the `web-https` resources exported by the web components to ensure ordering. The `web-https` resources are also passed in via the `balancermembers` parameter. This allows the load balancer to collect an number of HTTP resources.
```

Comment:
Passing as a parameter instead of 'consumes' allows the the load balancer to collect any number of Http[note capitalization] resources.